### ISSUE TITLE
It is hard to follow without examples

### ISSUE THREAD

**Issue by thalluripk** [MicrosoftDocs/power-platform#1493]
[Enter feedback here]
Without examples, it is very difficult to understand what to feed in the parameters. For example, I don't have a clue how I should feed the filter criteria. Is it OData filters? PowerShell filters?

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 10bda820-3ad2-29d7-9cb2-bc87afbd1f7b
* Version Independent ID: 10bda820-3ad2-29d7-9cb2-bc87afbd1f7b
* Content: [Microsoft Power Platform CLI org command group - Power Platform](https://learn.microsoft.com/en-us/power-platform/developer/cli/reference/org)
* Content Source: [power-platform/developer/cli/reference/org.md](https://github.com/MicrosoftDocs/power-platform/blob/main/power-platform/developer/cli/reference/org.md)
* Service: **power-platform**
* Sub-service: **developer**
* GitHub Login: @kkanakas
* Microsoft Alias: **kartikka**

**Comment by bishalgoswami** [comment]
Thanks for your comment, @thalluripk. This issue has been assigned to the doc author for review.

**Comment by JimDaly** [comment]
@thalluripk 
Thanks for your feedback. 
I'll work with the team to try and get some examples here.

**Comment by JimDaly** [comment]
@thalluripk 

We added an example for the `--filter` parameter. It is a regular expression match on the organization name.
(you might not be able to see it until publishing completes by EOD)

There is an on-going effort to add more examples here. If you find more cases where you need examples, please let us know.
